<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageFeesDialog</class>
 <widget class="QDialog" name="ManageFeesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fees Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>961</width>
     <height>661</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Schule</string>
    </attribute>
    <widget class="QLineEdit" name="lineSchuleFee">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>580</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>580</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>New Fee</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableCourses">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>911</width>
       <height>501</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushSaveSchuleFee">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>580</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboCourses">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>540</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>540</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Course</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Uni</string>
    </attribute>
    <widget class="QComboBox" name="comboUniCourses">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>540</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushSaveUniFee">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>580</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>New Fee</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Course</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableUniCourses">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>911</width>
       <height>461</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineUniFee">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>580</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Department</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDeps">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>500</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Sprache</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>580</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>New Fee</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushSaveSpracheFee">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>590</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Instutut</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>490</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Diploma</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableSpracheCourses">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>921</width>
       <height>471</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDiplomas">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>500</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboInstituten">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>580</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineSpracheFee">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>590</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboSprachen">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>540</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboScheduleDiplomas">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>550</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>540</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Schedule</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Discounts</string>
    </attribute>
    <widget class="QTableView" name="tableDiscounts">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>591</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>New Fee</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushSaveDiscount">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>370</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDiscountType">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>330</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNewDiscount">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
